jetty-ee10/jetty-ee10-demos/jetty-ee10-demo-spec/jetty-ee10-demo-container-initializer/src/main/java/org/example/initializer/FooInitializer.java

```diff
@@ -83,8 +83,8 @@ public void onStartup(Set<Class<?>> classes, ServletContext context)
         if (context.getAttribute("org.example.Foo") != null)
             throw new IllegalStateException("FooInitializer on Startup already called");
 
-        context.setAttribute("org.example.Foo", new ArrayList<Class>(classes));
-        ServletRegistration.Dynamic reg = context.addServlet("AnnotationTest", "org.example.AnnotationTest");
+        context.setAttribute("org.example.Foo", new ArrayList<>(classes));
+        ServletRegistration.Dynamic reg = context.addServlet("AnnotationTest", "org.example.test.AnnotationTest");
```

Comment:
Maybe we should refer to the class, rather than the name of the class?

Reply:
The `AnnotationTest` class is in the `jetty-ee10-demo-spec-webapp` project.
The above line in `FooInitializer` is in the `jetty-ee10-demo-container-initializer` project (It cannot see the `AnnotationTest` class)